### ISSUE TITLE
Bug with sum_working_zone_areas on all_in_one_sowing intervention recording

### ISSUE THREAD

**Issue by burisu** [ekylibre/ekylibre#613]
```
An ArgumentError occurred in interventions#compute:

  wrong number of arguments (1 for 0)
  lib/procedo/engine/functions.rb:45:in `sum_working_zone_areas'
```

**Comment by ionosphere** [comment]
I don't know if there's a link with the error message bellow ?

#<ActiveModel::Errors:0x0055e3ba14b9c0 @base=#<Intervention id: nil, issue_id: nil, prescription_id: nil, procedure_name: "all_in_one_sowing", state: "undone", started_at: nil, stopped_at: nil, created_at: nil, updated_at: nil, creator_id: nil, updater_id: nil, lock_version: 0, event_id: nil, number: "45", description: "", working_duration: 0, whole_duration: 0, actions: [:fertilization, :sowing], custom_fields: nil>, @messages={:"inputs.quantity_population"=>["Quantité population ne peut pas être vide"]}>

**Comment by PH64** [comment]
The problem seems solved in V2.0 RC 18
